README.md

```diff
@@ -1,2 +1,125 @@
-# My-Portfolio
-Building a Personal Portfolio to showcase skills
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 My Portfolio](#-my-portfolio)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 My Portfolio<a name="Creating Personal Portfolio project "></a>
+
+
+**My Portfolio** is a project build with the knowledge of Html and CSS.
+
+## 🛠 Built With <a name="HTML&CSS"></a>
+
+<details>
+### Key Features <a name="key-features"></a>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://w3schools.com">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://w3schools.com">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>LINTERS</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config">Linters</a></li>
+  </ul>
+</details>
+
+KEY Features
+- Portfolio Website to showcase my projects
+- Connect with Employers 
+
+
+
+## 💻 Getting Started <a name="https://github.com/shurichmond/My-Portfolio"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+ 
+ -VS code, Node.js and Git
+
+### Setup
+
+Clone this repository to your desired folder:
+
+-git clone https://github.com/shurichmond/My-Portfolio.git
+
+### Install
+
+Install this project with:
+
+  `npm install`
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Deployment
+
+You can deploy this project using:
+
+github
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1** Richmond Shu Nfortayong
+
+- GitHub: [@githubhandle](https://github.com/shurichmond)
+- Twitter: [@twitterhandle](https://twitter.com/shurichmond85)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shu-nfortayong-richmond-10a28620a/)
```

Comment:
The link to your `Twitter` account is broken broken, please kindly fix that, let's avoid broken links in our readme file for a professional repository. 💪🛠